sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuCast.scala

```diff
@@ -191,14 +236,13 @@ object GpuCast {
       input: ColumnView,
       fromDataType: DataType,
       toDataType: DataType,
-      ansiMode: Boolean,
-      legacyCastToString: Boolean,
-      stringToDateAnsiModeEnabled: Boolean): ColumnVector = {
-
+      options: CastOptions = DefaultCastOptions): ColumnVector = {
     if (DataType.equalsStructurally(fromDataType, toDataType)) {
       return input.copyToColumnVector()
     }
 
+    import options.ansiMode
```

Comment:
nit: this seems confusing to me. Would it be simpler to just assign to val perhaps?
```suggestion
    val ansiMode = options.ansiMode
```

Reply:
I can but what about in cases where I need to access more variables from it? Do you want me to create a separate val for each one?

Reply:
I see that the import is used later on to import multiple attributes, and it would be overkill to store each one in a separate variable, so maybe the import approach is the way to go.